srcpkgs/interception-tools/files/udevmon/run

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+if [ -r ./conf ]; then
+	. ./conf
+	if [ ! -z "${CONF_FILE}" ]; then
+		OPTS="${OPTS:+$OPTS }-c '${CONF_FILE}'"
```

Comment:
```suggestion
		OPTS="${OPTS} -c '${CONF_FILE}'"
```

is slightly simpler and the space doesn't hurt.

Reply:
This was how I did it at first, but then that perfectionist in me started to scream that there must be a way to not have that extra space there :) If you insist - I can change it though.
Also, there is actually only one supported command line argument `-c` (there is also `-h` for help, but we can skip that one). Do you think it makes sense to still have `OPTS` here? Maybe for the potential new arguments that might be added in the future releases?

Reply:
I'd rather use the simpler version ;)

And yes, any future args.

Reply:
Done.